<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemDefGroupForm</class>
 <widget class="QWidget" name="ItemDefGroupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Num. required groups</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leNumReqGroups"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Max. num. groups</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="leMaxNumGroups"/>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="cbExtensible">
     <property name="text">
      <string>Extensible</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Common subgroup label</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="leCommonLabel"/>
     </item>
     <item>
      <widget class="QCheckBox" name="cbCommonLabel">
       <property name="text">
        <string>Use</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
